### PR TITLE
Story 4 attribution

### PR DESCRIPTION
**Pull request by robertkieronreeves**
Added attribution tab next to the instructions tab on the main start page.

A list has been added with all credits with clickable links.

JS added to allow the user to toggle between the instructions and the attributions tabs.

### CODE REVIEW

**Review by patrickkenyon — DISMISSED** [review]

File: css/game.css [hunk 1]
```diff
@@ -73,7 +73,7 @@ body {
     float: left;
 }
 
-#splash-instructions {
+#text-box {
```
Comment (mporam) [inline css/game.css]: this is a terrible id name

File: js/start-game.js [hunk 2]
```diff
@@ -13,3 +13,13 @@ document.querySelector('#splash-open-game').addEventListener('click', function()
     listenCityClick()
     listenKeypress()
 })
+
+document.querySelector('#instructions-tab').addEventListener('click', function() {
```
Comment (mporam) [inline js/start-game.js:17]: no active state on these tabs, need to show the users which one they are on

**Review by mporam — CHANGES_REQUESTED** [review]

File: css/game.css (hunk 1 above)
Comment (robertkieronreeves) [reply css/game.css]: changed in 22106c3

File: js/start-game.js (hunk 2 above)
Comment (robertkieronreeves) [reply js/start-game.js:17]: changed in 22106c3

**Review by mporam — APPROVED** [review]

**Pull request closed without merging**